.github/dependabot.yml

```diff
@@ -13,6 +13,9 @@ updates:
       - dependency-name: jakarta.xml.ws:jakarta.xml.ws-api
         versions:
           - ">= 3.0.0"
+    # Prevent PRs from opening except for security updates, given Renovate
+    # is used to manage regular dependency updates
+    open-pull-requests-limit: 0
```

Comment:
I wonder if we should disable dependabot altogether? Otherwise we might have to manage exclusions in two ways?

Reply:
The `open-pull-requests-limit: 0` setting will prevent dependabot from opening PRs for any regular version updates.  It will not, however, prevent security updates from coming through.   I'm not entirely sure whether we'll end up with two PRs for each security update (one from renovate and one from dependabot).  I haven't been able to find much in terms of documentation on this topic, other than at the link below:

https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts
"Renovate can read GitHub's Vulnerability Alerts to customize its Pull Requests. For this to work, you must enable the [Dependency graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#enabling-the-dependency-graph), and [Dependabot alerts](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-security-and-analysis-settings-for-your-repository)."

My thoughts are we keep both dependabot and renovate running for now, and make a call on disabling dependabot altogether once we see how things play out.  Worst case, we will end up with some duplicate PRs for security updates, but these should be very infrequent.